src/img/items/sweet-potato.piskel

```diff
@@ -0,0 +1 @@
+{"modelVersion":2,"piskel":{"name":"New Piskel","description":"","fps":12,"height":24,"width":24,"layers":["{\"name\":\"Layer 1\",\"opacity\":1,\"frameCount\":1,\"chunks\":[{\"layout\":[[0]],\"base64PNG\":\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAA7UlEQVRIS2NkoDFgpLH5DKMWEAxhioMoyoD3P7otyy58hptLlgXIhjYHaYLNv3rnMoO2ii6YXbvuOgPMEpIsgBkMMxRX+JBsAbEGwywkyQKQ4bhcDAoWGIAFD0lBhM1wXIYiBxdRPsBlOLJLKYoDdAuQUwmhxE/QB8QYjmwhriQKcghGMiXGcHw+QHY9hgUww3EFB6FgQjccxQKQ4Z0FaQy/Hh0mFMRY5bEZTpEF6L4hyQJ0zfiCBmQwCCAXcMheRIlkfLkWW7jgcjVOC0AS2EpK9FwK4+NyNV4LkCUJlfXEpAaSimtiDERXQ3MLANJApBnOiL6/AAAAAElFTkSuQmCC\"}]}"]}}
```

Comment:
This has `"name":"New Piskel"`, and it looks like it should be `"name":"sweet-potato"`.